backend/app/services/chat/trigger/unified.py

```
@@ -375,36 +376,76 @@ def _build_cloud_gateway_model_config(
+            model_type,
+            namespace,
+            resource_user_id,
+        )
+        kind = (
+            db.query(Kind)
+            .filter(
+                Kind.user_id == resource_user_id,
+                Kind.kind == "Model",
+                Kind.namespace == namespace,
+                Kind.name == model_name,
+                Kind.is_active == True,
```

Ruff (0.16.1)

[error] Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)